applications/stream-applications-core/pom.xml

```diff
@@ -4,31 +4,31 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.springframework.cloud.stream.app</groupId>
     <artifactId>stream-applications-core</artifactId>
-    <version>3.0.0-SNAPSHOT</version>
+    <version>3.1.0-SNAPSHOT</version>
     <name>stream-applications-core</name>
     <description>Stream Applications Core Parent</description>
     <packaging>pom</packaging>
 
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.3.4.RELEASE</version>
+        <version>2.4.0-SNAPSHOT</version>
         <relativePath/>
     </parent>
 
     <properties>
         <apps.base-image>springcloud/baseimage:1.0.0</apps.base-image>
         <java.version>1.8</java.version>
-        <spring-boot.version>2.3.4.RELEASE</spring-boot.version>
+        <spring-boot.version>2.4.0-SNAPSHOT</spring-boot.version>
         <stream-apps-core.version>${project.version}</stream-apps-core.version>
         <java-functions.version>1.0.0-SNAPSHOT</java-functions.version>
         <spring-cloud-dataflow-apps-generator-plugin.version>1.0.0-M1</spring-cloud-dataflow-apps-generator-plugin.version>
         <spring-cloud-dataflow-apps-docs-plugin.version>1.0.0-M1</spring-cloud-dataflow-apps-docs-plugin.version>
-        <spring-cloud-stream.version>3.0.9.BUILD-SNAPSHOT</spring-cloud-stream.version>
-        <spring-cloud-function-dependencies.version>3.0.9.RELEASE</spring-cloud-function-dependencies.version>
-		<spring-integration-dependencies.version>5.3.3.BUILD-SNAPSHOT</spring-integration-dependencies.version>
+        <spring-cloud-stream.version>3.1.0-SNAPSHOT</spring-cloud-stream.version>
+        <spring-cloud-function-dependencies.version>3.1.0-SNAPSHOT</spring-cloud-function-dependencies.version>
+		    <spring-integration-dependencies.version>5.4.0-SNAPSHOT</spring-integration-dependencies.version>
```

Comment:
Please , fix this indent. I don't think that SI deserves its own special view over here 😄